File: EnumItemsSourceBehaviorExample/Common/FilterStringConverterExtension.cs
using System;
using System.Linq;
using System.Windows.Markup;
using System.Collections.Generic;

namespace EnumItemsSourceBehaviorExample.Common {
    public class FilterStringConverterExtension : MarkupExtension {
        public override object ProvideValue(IServiceProvider serviceProvider) {
            return new FilterStringConverter();
        }
    }
}

File: VB/EnumItemsSourceBehaviorExample/Common/FilterStringConverterExtension.vb
Imports System
Imports System.Windows.Markup

Namespace EnumItemsSourceBehaviorExample.Common

    Public Class FilterStringConverterExtension
        Inherits MarkupExtension

        Public Overrides Function ProvideValue(ByVal serviceProvider As IServiceProvider) As Object
            Return New FilterStringConverter()
        End Function
    End Class
End Namespace
